src/datamodel/turn/test/test_turn.rs

```diff
@@ -0,0 +1,71 @@
+use crate::datamodel::quest::{FinishedQuest, Quest, QuestObjective};
+use crate::datamodel::turn::Turn;
+
+use crate::datamodel::character::{Merlin, Mordred, Percival};
+use crate::datamodel::turn::finished_turn::FinishedTurn;
+use crate::datamodel::turn::turn::TurnRole;
+use crate::datamodel::turn::turn::TurnRole::{ProposalVoter, QuestProposer};
+use crate::datamodel::turn::turn_history::TurnHistory;
+use crate::datamodel::Player;
+use std::collections::HashMap;
+
+#[test]
+fn test_turn_construction() {
+    let (jimi, pato, volan) = get_players();
+
+    let mut map: HashMap<String, TurnRole> = HashMap::new();
+
+    map.insert(jimi.name.clone(), ProposalVoter);
+    map.insert(pato.name.clone(), ProposalVoter);
+    map.insert(volan.name.clone(), ProposalVoter);
+
+    map.insert(pato.name.clone(), QuestProposer);
+
+    let _turn = Turn {
+        current_turn_roles: map,
+        current_quest_objective: QuestObjective {
+            num_players: 3,
+            required_successes: 3,
+        },
+    };
+
+    assert!(true);
+}
+
+#[test]
+fn test_turn_history_construction() {
+    let (jimi, pato, volan) = get_players();
+    let _turn_history = TurnHistory {
+        finished_turns: vec![FinishedTurn {
+            quest: FinishedQuest {
+                won: false,
+                quest: Quest {
+                    objective: QuestObjective {
+                        num_players: 2,
+                        required_successes: 2,
+                    },
+                    proposer: &pato,
+                    members: vec![&jimi, &volan],
+                },
+            },
+        }],
+    };
+
+    assert!(true);
```

Comment:
You don't need to add this line. In fact, if `panic!` macro isn't called throughout the test's lines the test will success. 
So, please remove it.